quipucords/fingerprinter/task.py

```diff
@@ -123,14 +126,30 @@ def run(self, manager_interrupt):
         fact_collection = self.scan_task.fact_collection
 
         # remove results from previous interrupted scan
-        existing_fingerprints = SystemFingerprint.objects.filter(
-            report_id=fact_collection.id)
-        if existing_fingerprints:
+        deployment_report = fact_collection.deployment_report
+        if not deployment_report:
+            deployment_report = DeploymentReport()
+            deployment_report.save()
+
+            # Set the report ids.  Right now they are deployemnt report id
+            # but they do not have to
+            deployment_report.report_id = deployment_report.id
+            deployment_report.save()
+
+            # Set the report ids.  Right now they are deployemnt report id
```

Comment:
This seems like a repeat comment, remove one of them and change `deployemnt` to `deployment`